web/src/components/OpenEndedQuestion/OpenEndedQuestion.tsx

```diff
@@ -21,13 +21,15 @@ const OpenEndedQuestion = ({
         <Square size="20px" bg="grayIcon" data-testid="square" />
         <Text data-testid="question">{question}</Text>
       </Stack>}
-      <Input
+      <Box w='100%' pl={4}>
```

Comment:
What is the goal of this left padding ?

<img width="419" alt="Capture d’écran 2023-05-30 à 09 01 17" src="https://github.com/x-image-privacy/privacyAdvisor/assets/39373170/ac81b066-d03c-485b-8fa9-c2686d0dfdc3">

If you want to have the square be the only element to the left and make it so the field and the text of the question are alinged you should change the structure and have first a horizontal stack to put the square to the left and then put your question and field inside a vertical stack.

Reply:
![image](https://github.com/x-image-privacy/privacyAdvisor/assets/43374563/ad3797e9-4c07-40f0-8138-108a9cb571c0)

It is for aligning the input with the question text and not the square (as in like scale question field)

Reply:
If you want to align, I would say, A is better.